ruby/ruby_benchmarks.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Ruby trunk
+
+# Build base image for Ruby benchmarks
+cd ruby_trunk/ruby_benchmarks
+sudo docker build --no-cache -t rubybench/ruby_trunk .
```

Comment:
The script should just run the benchmarks. Don't build the image here. Building should be handled in a separate script